### DISCUSSION TITLE
Upgrading Teslamate - No Tire pressure reading or Grafana update

### DISCUSSION THREAD

**Discussion by maxhac03** [teslamate-org/teslamate#2723] [General]
Docker installation running 1.26.1

I can see that tire pressure and a more recent version of grafana has been added but even after doing the upgrade i still don't see the tire pressures and my Grafana install is still on v8.3.4.

The docker images has been updated 3 days ago (https://hub.docker.com/u/teslamate) and i'm using the normal procedure to do the upgrade (docker-compose pull and docker-compose up -d)

Is there something broken on my side or am i doing something wrong? I can do Postgres upgrades but Grafana is not upgrading.

**Comment by cwanja** [comment]
What Docker image are you on? Latest would only include published releases and the tire pressure is not yet released. I would suspect a release is coming, as the CHANGELOG was updated. But only Adrian knows specifically when.

**Reply by maxhac03** [reply]
https://docs.teslamate.org/docs/guides/traefik

This confirm what i was thinking. I saw that the images got updated and i thought that simply doing an upgrade would add the features.

Thanks!

**Reply by cwanja** [reply]
You can change the image in the YML to `edge`, but you might also experience some bugs along the way. Happy to help!
